models/unit_tests.go

```diff
@@ -48,6 +48,7 @@ func MainTest(m *testing.M, pathToGiteaRoot string) {
 	setting.RunUser = "runuser"
 	setting.SSH.Port = 3000
 	setting.SSH.Domain = "try.gitea.io"
+	setting.UseSQLite3 = true
```

Comment:
Does this line need to be added? I'm just concerned with what might happen if Gitea isn't compiled with sqlite support.

Reply:
This is only for the tests (which normally are done with sqlite support). As we need slightly different queries for different database types, I need to check it. Do you have a better idea?

Reply:
That seems reasonable. In that case I'm ok with this. I'll leave this as "unresolved" just so others can see it in case they have alternate solutions.